phpBB/includes/acp/acp_profile.php

```diff
@@ -735,12 +758,12 @@ function main($id, $mode)
 				'FIELD_TYPE'		=> $profile_field->get_name(),
 
 				'L_ACTIVATE_DEACTIVATE'		=> $user->lang[$active_lang],
-				'U_ACTIVATE_DEACTIVATE'		=> $this->u_action . "&amp;action=$active_value&amp;field_id=$id",
+				'U_ACTIVATE_DEACTIVATE'		=> $this->u_action . "&amp;action=$active_value&amp;field_id=$id" . '&amp;hash=' . generate_link_hash('acp_profile'),
 				'U_EDIT'					=> $this->u_action . "&amp;action=edit&amp;field_id=$id",
```

Comment:
Editing is failing
https://tracker.phpbb.com/browse/PHPBB3-14789?focusedCommentId=59497&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-59497